front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/HomepageBanner/index.tsx

```diff
@@ -93,9 +95,15 @@ type Props = {
 };
 
 const HomepageBanner = ({ homepageSettings, image }: Props) => {
+  const { data: authUser } = useAuthUser();
   const [search] = useSearchParams();
-
-  return search.get('variant') === 'signedIn' ? (
+  const editor = useEditor();
+  const isEditorInPreviewMode =
+    (editor.store.getState() as any)?.options.enabled === false;
```

Comment:
The types that come with the library for this particular method are incorrect or outdated so I needed to cast as any to make it work.